fetch.bs

```diff
@@ -917,6 +919,16 @@ not always relevant and might require different behavior.
    <td>"<code>manifest</code>"
    <td><code>manifest-src</code>
    <td>HTML's <code>&lt;link rel=manifest></code>
+  <tr>
+   <td>"<code>prefetch</code>"
+   <td>""
```

Comment:
rowspan=2 for this td and the next? And then remove the from the next row?

Reply:
updated. Let me know if this is what you had in mind